### PR TITLE
Add basic proposal xls importer

### PR DESCRIPTION
**Pull request by beagleknight**
# What and Why

We want to make a initial setup for proposals but we don't want to store them as seeds or as a file in the repository. I have created a simple rake task to import a xlsx file from a url. The task parses the xlsx file and create `oficial` proposals into the database.
The same task create `categories` and `subcategories` as needed.

As an extra bonus I have included a `categories:import` and `categories:export` tasks to manage a json file with categories. This file could be used to update categories after filling translations in the json file.
# QA

Run `rake proposals:import[url]` where `url` corresponds to a xlsx file.
# GIF Tax

# TODOs
- [x] Parse xls file
- [x] Parse district 
- [x] Parse categories and subcategories
- [x] Import proposals
- [x] Check `acts_as_list` behavior
- [x] Subcategory sense category?
- [x] Author?
- [x] Responsible name?
- [x] Create rake task to re-generate categories.yml
- [ ] Tests

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,7 +53,6 @@ gem 'devise-async'
 gem 'newrelic_rpm', '~> 3.14'
 gem 'whenever', require: false
 gem 'pg_search'
-gem 'acts_as_list'
```
Comment (andreslucena) [inline Gemfile:56]: We didn't use this finally, right?

Comment (beagleknight) [inline Gemfile:56]: @andreslucena Nope! We thought it's not useful right now because we are just using the order by position and not manipulating lists.

**Comment by beagleknight** [comment]
@josepjaume Before merging this I want to add some tests for both `CategoryImporter` and `ProposalXLSImporter`. Stay tuned!

**Comment by josepjaume** [comment]
MEGA GIFFFFF

:boom: :boom:

**Pull request merged**